scripts/observers/config.py

```diff
@@ -125,6 +141,47 @@ def prepare_seed_node(working_dir):
     os.chmod(working_dir/"seednode/seednode", st.st_mode | stat.S_IEXEC)
 
 
+def prepare_proxy(working_dir):
```

Comment:
[question]

Why do you have to "manually" prepare the proxy binary instead of setting `USE_PROXY`  from `mx-chain-go` repo to true?

Reply:
because in order to can set `USE_PROXY` I have also to add observers in my local testnet.  
If a add observers my number of nodes will double.

Reply:
Right, good observation